config_in_cmake.h

```diff
@@ -21,6 +21,9 @@
 /* Define this to use AES-GCM. */
 #cmakedefine GCM 1
 
+/* Define this to use MBEDTLS. */
+#cmakedefine MBEDTLS 1
```

Comment:
I would reorder these to put `MBEDTLS` just after `OPENSSL`.

Reply:
Sure, I will change the order.